{
  "uid" : "d1640b7330103f51",
  "name" : "Проверить сообщение об отправке ссылки восстановления пароля на емайл",
  "fullName" : "tests.test_password_recovery.test_password_recovery.TestPasswordRecoveryPage#test_fill_correct_email",
  "historyId" : "31cba4394a235068db88fa9e5b27aa06",
  "time" : {
    "start" : 1714273895523,
    "stop" : 1714273895523,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55d1cbdc1e33 <unknown>\n#1 0x55d1cbab1567 <unknown>\n#2 0x55d1cbafc27d <unknown>\n#3 0x55d1cbafc331 <unknown>\n#4 0x55d1cbb3f624 <unknown>\n#5 0x55d1cbb1e38d <unknown>\n#6 0x55d1cbb3cae2 <unknown>\n#7 0x55d1cbb1e103 <unknown>\n#8 0x55d1cbaef1ba <unknown>\n#9 0x55d1cbaefb8e <unknown>\n#10 0x55d1cbd8659b <unknown>\n#11 0x55d1cbd8a34b <unknown>\n#12 0x55d1cbd72681 <unknown>\n#13 0x55d1cbd8aeb2 <unknown>\n#14 0x55d1cbd576af <unknown>\n#15 0x55d1cbdb1428 <unknown>\n#16 0x55d1cbdb1600 <unknown>\n#17 0x55d1cbdc0f84 <unknown>\n#18 0x7f2856094ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"bb74b2bc8e650e5ae34ad684f40145e7\")>\n\n    @pytest.fixture(scope='function')\n    def recovery_page_open(driver):\n        recovery_page = PasswordRecoveryPage(driver)\n        driver.get(Constant.PASSWORD_RECOVERY)\n>       recovery_page.get_allow_all_cookies().click()\n\ntests/test_password_recovery/conftest.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/password_recovery_page.py:38: in get_allow_all_cookies\n    return self.element_is_visible(self.locator.ALLOW_ALL_COOKIES)\npages/base_page.py:19: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:35: in element_is_present\n    return Wait(self.driver, self.timeout).until(es.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"bb74b2bc8e650e5ae34ad684f40145e7\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7f3f3a4d9e40>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55d1cbdc1e33 <unknown>\nE       #1 0x55d1cbab1567 <unknown>\nE       #2 0x55d1cbafc27d <unknown>\nE       #3 0x55d1cbafc331 <unknown>\nE       #4 0x55d1cbb3f624 <unknown>\nE       #5 0x55d1cbb1e38d <unknown>\nE       #6 0x55d1cbb3cae2 <unknown>\nE       #7 0x55d1cbb1e103 <unknown>\nE       #8 0x55d1cbaef1ba <unknown>\nE       #9 0x55d1cbaefb8e <unknown>\nE       #10 0x55d1cbd8659b <unknown>\nE       #11 0x55d1cbd8a34b <unknown>\nE       #12 0x55d1cbd72681 <unknown>\nE       #13 0x55d1cbd8aeb2 <unknown>\nE       #14 0x55d1cbd576af <unknown>\nE       #15 0x55d1cbdb1428 <unknown>\nE       #16 0x55d1cbdb1600 <unknown>\nE       #17 0x55d1cbdc0f84 <unknown>\nE       #18 0x7f2856094ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1714270943308,
      "stop" : 1714270943308,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1714270943291,
      "stop" : 1714270943307,
      "duration" : 16
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1714273895524,
      "stop" : 1714273895722,
      "duration" : 198
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "recovery_page_open",
    "time" : {
      "start" : 1714273895722,
      "stop" : 1714273916456,
      "duration" : 20734
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55d1cbdc1e33 <unknown>\n#1 0x55d1cbab1567 <unknown>\n#2 0x55d1cbafc27d <unknown>\n#3 0x55d1cbafc331 <unknown>\n#4 0x55d1cbb3f624 <unknown>\n#5 0x55d1cbb1e38d <unknown>\n#6 0x55d1cbb3cae2 <unknown>\n#7 0x55d1cbb1e103 <unknown>\n#8 0x55d1cbaef1ba <unknown>\n#9 0x55d1cbaefb8e <unknown>\n#10 0x55d1cbd8659b <unknown>\n#11 0x55d1cbd8a34b <unknown>\n#12 0x55d1cbd72681 <unknown>\n#13 0x55d1cbd8aeb2 <unknown>\n#14 0x55d1cbd576af <unknown>\n#15 0x55d1cbdb1428 <unknown>\n#16 0x55d1cbdb1600 <unknown>\n#17 0x55d1cbdc0f84 <unknown>\n#18 0x7f2856094ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 917, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/tests/test_password_recovery/conftest.py\", line 10, in recovery_page_open\n    recovery_page.get_allow_all_cookies().click()\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/pages/password_recovery_page.py\", line 38, in get_allow_all_cookies\n    return self.element_is_visible(self.locator.ALLOW_ALL_COOKIES)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/pages/base_page.py\", line 19, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/pages/base_page.py\", line 35, in element_is_present\n    return Wait(self.driver, self.timeout).until(es.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Проверка видимости кнопки 'Принимаю все' в сообщении о принятии файлов cookie",
      "time" : {
        "start" : 1714273896048,
        "stop" : 1714273916456,
        "duration" : 20408
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55d1cbdc1e33 <unknown>\n#1 0x55d1cbab1567 <unknown>\n#2 0x55d1cbafc27d <unknown>\n#3 0x55d1cbafc331 <unknown>\n#4 0x55d1cbb3f624 <unknown>\n#5 0x55d1cbb1e38d <unknown>\n#6 0x55d1cbb3cae2 <unknown>\n#7 0x55d1cbb1e103 <unknown>\n#8 0x55d1cbaef1ba <unknown>\n#9 0x55d1cbaefb8e <unknown>\n#10 0x55d1cbd8659b <unknown>\n#11 0x55d1cbd8a34b <unknown>\n#12 0x55d1cbd72681 <unknown>\n#13 0x55d1cbd8aeb2 <unknown>\n#14 0x55d1cbd576af <unknown>\n#15 0x55d1cbdb1428 <unknown>\n#16 0x55d1cbdb1600 <unknown>\n#17 0x55d1cbdc0f84 <unknown>\n#18 0x7f2856094ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/pages/password_recovery_page.py\", line 38, in get_allow_all_cookies\n    return self.element_is_visible(self.locator.ALLOW_ALL_COOKIES)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/pages/base_page.py\", line 19, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/pages/base_page.py\", line 35, in element_is_present\n    return Wait(self.driver, self.timeout).until(es.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1714273916471,
      "stop" : 1714273916529,
      "duration" : 58
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Тестирование страницы 'Восстановление пароля'"
  }, {
    "name" : "tag",
    "value" : "regress"
  }, {
    "name" : "parentSuite",
    "value" : "tests.test_password_recovery"
  }, {
    "name" : "suite",
    "value" : "test_password_recovery"
  }, {
    "name" : "subSuite",
    "value" : "TestPasswordRecoveryPage"
  }, {
    "name" : "host",
    "value" : "fv-az1383-514"
  }, {
    "name" : "thread",
    "value" : "2368-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_password_recovery.test_password_recovery"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "browser",
    "value" : "'chrome'"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 1,
        "skipped" : 0,
        "passed" : 11,
        "unknown" : 0,
        "total" : 12
      },
      "items" : [ {
        "uid" : "782e474e4de2e1d7",
        "reportUrl" : "https://Nat754.github.io/PulseWave/7//#testresult/782e474e4de2e1d7",
        "status" : "passed",
        "time" : {
          "start" : 1714185214053,
          "stop" : 1714185215749,
          "duration" : 1696
        }
      }, {
        "uid" : "fb907f5484969aec",
        "reportUrl" : "https://Nat754.github.io/PulseWave/6//#testresult/fb907f5484969aec",
        "status" : "passed",
        "time" : {
          "start" : 1714148640581,
          "stop" : 1714148642281,
          "duration" : 1700
        }
      }, {
        "uid" : "2f508f624ce92b97",
        "reportUrl" : "https://Nat754.github.io/PulseWave/6//#testresult/2f508f624ce92b97",
        "status" : "passed",
        "time" : {
          "start" : 1714098690597,
          "stop" : 1714098692285,
          "duration" : 1688
        }
      }, {
        "uid" : "595fe6df1cda825f",
        "reportUrl" : "https://Nat754.github.io/PulseWave/5//#testresult/595fe6df1cda825f",
        "status" : "passed",
        "time" : {
          "start" : 1714014545638,
          "stop" : 1714014547331,
          "duration" : 1693
        }
      }, {
        "uid" : "1305d54b4f8a33ff",
        "reportUrl" : "https://Nat754.github.io/PulseWave/211//#testresult/1305d54b4f8a33ff",
        "status" : "passed",
        "time" : {
          "start" : 1713946053451,
          "stop" : 1713946055133,
          "duration" : 1682
        }
      }, {
        "uid" : "ddbe594a39d73106",
        "reportUrl" : "https://Nat754.github.io/PulseWave/4//#testresult/ddbe594a39d73106",
        "status" : "passed",
        "time" : {
          "start" : 1713925843859,
          "stop" : 1713925845543,
          "duration" : 1684
        }
      }, {
        "uid" : "c980b52fcd361f90",
        "reportUrl" : "https://Nat754.github.io/PulseWave/211//#testresult/c980b52fcd361f90",
        "status" : "passed",
        "time" : {
          "start" : 1713901719313,
          "stop" : 1713901721009,
          "duration" : 1696
        }
      }, {
        "uid" : "e69861d980a15ac",
        "reportUrl" : "https://Nat754.github.io/PulseWave/210//#testresult/e69861d980a15ac",
        "status" : "passed",
        "time" : {
          "start" : 1713885952877,
          "stop" : 1713885954558,
          "duration" : 1681
        }
      }, {
        "uid" : "bc32672695e52b89",
        "reportUrl" : "https://Nat754.github.io/PulseWave/209//#testresult/bc32672695e52b89",
        "status" : "passed",
        "time" : {
          "start" : 1713883842432,
          "stop" : 1713883844129,
          "duration" : 1697
        }
      }, {
        "uid" : "c1b7d9d297c22e6c",
        "reportUrl" : "https://Nat754.github.io/PulseWave/209//#testresult/c1b7d9d297c22e6c",
        "status" : "passed",
        "time" : {
          "start" : 1713882370569,
          "stop" : 1713882372259,
          "duration" : 1690
        }
      }, {
        "uid" : "5ab9a58a9c428a91",
        "reportUrl" : "https://Nat754.github.io/PulseWave/208//#testresult/5ab9a58a9c428a91",
        "status" : "passed",
        "time" : {
          "start" : 1713879549809,
          "stop" : 1713879551498,
          "duration" : 1689
        }
      } ]
    },
    "tags" : [ "regress" ]
  },
  "source" : "d1640b7330103f51.json",
  "parameterValues" : [ "'chrome'" ]
}